src/EasyAdminBundle.php

```diff
@@ -29,7 +30,13 @@ public function build(ContainerBuilder $container)
     {
         $container->addCompilerPass(new EasyAdminFormTypePass(), PassConfig::TYPE_BEFORE_REMOVING);
         $container->addCompilerPass(new EasyAdminConfigPass());
-        $container->addCompilerPass(new AnnotatedRouteControllerLoaderPass());
+        if (!class_exists(ServiceLocator::class)) { // DI < 3.3, so FrameworkBundle < 3.4 due to our conflict rules and its
```

Comment:
Do we then really need the conflict rule? Couldn't we add an `else` instead an use compiler pass priorities there for DI >= 3.3?

Reply:
Indeed we don't. PR updated :) (Priorities have been introduced in 3.2)